core/assets/bundles/bundle_de.properties

```diff
@@ -1146,7 +1146,7 @@ block.container.name = Behälter
 block.launch-pad.name = Launchpad
 block.launch-pad-large.name = Großes Launchpad
```

Comment:
Abschussrampe would be the correct translation

Reply:
Ic finde Launchpad persönlich besser, außerdem wurde launchpad [hier](https://github.com/Anuken/Mindustry/commit/75451743aaf76c26deceeb59717af81756b8666b) hinzugefügt. Das war vor 14 Monaten. Wenn du das ändern willst mache lieber selber einen PR auf.